clang/test/CodeGenCXX/arm64-generated-fn-attr.cpp

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_cc1 -triple aarch64-none-none -mbranch-target-enforce -msign-return-address=all -fcxx-exceptions -fexceptions -emit-llvm %s -o - | FileCheck %s --check-prefixes=CHECK
```

Comment:
perhaps it's time to add CodeGenCXX/AArch64/ ? We have more target-specific directories under CodeGen.

Then the weird `arm64-` prefix (not recommended for ELF/Arm official) can be removed.

You can remove `-fcxx-exceptions -fexceptions` since they are unused`. `-triple aarch64-none-none` can just be `-triple aarch64`.

Reply:
> perhaps it's time to add CodeGenCXX/AArch64/ ? We have more target-specific directories under CodeGen.
> Then the weird arm64- prefix (not recommended for ELF/Arm official) can be removed.

I'll create a NFC patch that cleans all of these up. (also in CodeGen where we have directories and prefixed files too.) 

> You can remove -fcxx-exceptions -fexceptions since they are unused

`__clang_call_terminate` is only generated when the exceptions are enabled. Clang generates the definition for it too and the test checks the right attributes for it.

> -triple aarch64-none-nonecan just be-triple aarch64`.

ack